src/components/profiles/wallet-base.vue

```diff
@@ -98,6 +98,12 @@ export default {
     },
     isOwner () {
       return this.username === this.account
+    },
+    tokenName () {
+      return this.$store.state.dao.settings.settings_pegTokenName_s ? this.$store.state.dao.settings.settings_pegTokenName_s : 'HUSD'
```

Comment:
@evgenibir `HUSD` should not be string

Reply:
@Alex-IS for now, in order not to waste time, I made it a constant, but in the future it would be possible to add this on the backend in the dao settings as a default token if none is specified